pages/blog/_posts/2020-05-18-formatting-erlang.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: blogpost
+permalink: /blog/choosing-erlang-formatter/
+title: Choosing an Erlang formatter
+date: 2020-05-18
+tags: erlang rebar3 coding culture
+author: <a href="https://www.linkedin.com/in/sebastian-weddmark-olsson/">Sebastian Weddmark Olsson</a>
+---
+
+I've looked into different formatters for Erlang and there are some
+options. The main two alternatives being `steamroller` and
+`rebar3_format`. `steamroller` is slow, which one could have imageined
+by its name. `rebar3_format` have issues with macros due to underlying
+dependencies.
+
+I've also briefly looked into an Erlang linter called `elvis`.
+
```

Comment:
If I understand right, this is the "intro" part? I think a more generic intro will sound better. I notice the other posts we have on the website give a general intro to kinda set the scene. I tried to rephrase a bit the first sentence, see if it makes sense? 
Eg: _There are many different Erlang formatters and I spent some time to look into what are the options that we can use at Working Group Two. The main two alternatives being `steamroller` and
 `rebar3_format`. `steamroller` is slow, which one could have imageined
 by its name. `rebar3_format` have issues with macros due to underlying
 dependencies.

 I've also briefly looked into an Erlang linter called `elvis`._